### ISSUE TITLE
IE9 and below - Requests not sending Cookie when doing polling handshake

### ISSUE THREAD

**Issue by jondubois** [socketio/engine.io#197]
I'm not sure if this is related to my project or if engine.io is in fact the root cause for this issue but I noticed that the 'io' cookie which is set during a handshake (using the 'Set-Cookie' response header) is not included in subsequent requests to the server (inside the 'Cookie' request header) - More importantly; any other cookies which are active on the client are not sent to the server either.

Basically I'm using an ssid cookie to synchronize multiple sockets (across multiple tabs in the browser) which belong to the same session.

This only appears to affect IE9 and below.
